client/packages/invoices/src/StockOut/api/operations.graphql

```diff
@@ -27,6 +27,12 @@ fragment StockOutLine on InvoiceLineNode {
     doses
   }
 
+  itemVariant {
```

Comment:
added item variant to stock out line - so the correct num doses is shown in the detail view!